modules/DMZ/data_kit/process_folder.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image  # image operations
+import os     # operations on directories
+import imghdr # image file recognition
+import numpy  # numpy arrays
```

Comment:
Eh, in @telelu03's defense, the comments dictate more what is being used from these imports than what is in these imports...ish